internal/controller/humiocluster_defaults.go

```diff
@@ -236,6 +236,7 @@ func NewHumioNodeManagerFromHumioNodePool(hc *humiov1alpha1.HumioCluster, hnp *h
 			PodLabels:                      hnp.PodLabels,
 			UpdateStrategy:                 hnp.UpdateStrategy,
 			PriorityClassName:              hnp.PriorityClassName,
+			NodePoolFeatures:               hnp.NodePoolFeatures,
```

Comment:
Did you intentionally leave this addition out of the func `NewHumioNodeManagerFromHumioCluster`?

Reply:
I did not. Fixed.